src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,14 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.NotNull;
 
 public class MovieSessionRequestDto {
+    @NotNull
     private Long movieId;
```

Comment:
maybe there's more reason to apply something like @Min(1) instead of @NotNull, because it has more constraints